src/main/java/com/phonebookservice/controller/ContactController.java

```diff
@@ -0,0 +1,55 @@
+package main.java.com.phonebookservice.controller;
+
+import main.java.com.phonebookservice.exception.PhoneBookControllerException;
+import main.java.com.phonebookservice.model.Contact;
+import main.java.com.phonebookservice.model.Model;
+import main.java.com.phonebookservice.server.DBIf;
+import main.java.com.phonebookservice.util.Messages;
+import main.java.com.phonebookservice.util.Util;
+
+public class ContactController extends AbstractController {
+
+    /**
+     * contact controller constructor.
+     *
+     * @param databse the database
+     */
+    public ContactController(final DBIf databse) {
+        super(databse);
+    }
+
+    @Override
+    public final void create(final Model model) {
```

Comment:
Java Doc please + Time complixity of this method.

Reply:
do I need to add a Javadoc to the override method?  All I know that Javadoc will be generated based on the superclasses Javadoc.

Reply:
yes but each method has it is own implmentation. 

for instance 
create in ContactController may takes O(n), where create in another controller may take O (N^2). you need to exlain what does this method doing